### ISSUE TITLE
[Question] Custom Dynamic Virtual Channel

### ISSUE THREAD

**Issue by acneidert** [Devolutions/IronRDP#61]
Do you have the possibility to create a Custom Dynamic Virtual Channel? 
What would be the way?

**Comment by awakecoding** [comment]
We do not implement either the static virtual channel or dynamic virtual channel APIs in IronRDP yet, and it's not planned at this point. Do you have a custom virtual channel extension which you would like to port over to IronRDP? Just to check, what you're [looking for is this](https://learn.microsoft.com/en-us/windows/win32/termserv/dvc-client-apis)?

**Comment by acneidert** [comment]
Sim! Eu gostaria de algo parecido como o [SocksOverRDP](https://github.com/nccgroup/SocksOverRDP). Basicamente eu gostaria de usar o RDP como um Proxy de uma porta do server para o cliente

**Comment by CBenoit** [comment]
I’ll work on that soon
